web/src/event-listeners/index.ts

```diff
@@ -177,9 +185,20 @@ export default function setupEventListeners(
           )
           if (childrenActionHashes.length) {
             event.stopPropagation()
-            if (childrenActionHashes.length === 1)
+            const keyboardNavPreference = getKeyboardNavigationPreference(state)
+            if (childrenActionHashes.length === 1) {
               panAndZoom(childrenActionHashes[0])
-            else {
+            } else if (keyboardNavPreference === COORDINATES) {
+              // navigate to the left-most child
+              const leftMostChild = _.minBy(
+                childrenActionHashes,
+                (actionHash) => {
+                  return state.ui.layout.coordinates[actionHash].x
+                }
+              )
+
+              panAndZoom(leftMostChild)
```

Comment:
since this logic is duplicate, perhaps we could move it into a single function. the list of hashes would be an argument